src/components/profile-view/profile-view.jsx

```diff
@@ -23,21 +33,26 @@ export const ProfileView = ({ user, movies, setUser, toggleFav }) => {
   // get Token
   const token = localStorage.getItem('token');
 
-  // Update user info
+
+  // Update user info / HANDLE UPDATE
   const handleUpdate = (event) => {
     // this prevents the default behavior of the form which is to reload the entire page
     event.preventDefault();
 
     const user = JSON.parse(localStorage.getItem('user'));
+    console.log(user);
 
     const data ={
       Username: username,
-      Password: password,
+      // Password: password,
       Email: email,
-      BirthDay: Birthday
     }
 
-    fetch(`https://myflix-api-qeb7.onrender.com/users/${user.Username}`, {
+    if (birthday) {
+      data.BirthDay = new Date(birthday).toISOString().split('T')[0]; // Convert birthday in date object
```

Comment:
@manjou ,  This is the reason birthday is not working. Your API expects `Birthday` not `BirthDay`.  You can see that the API expects it here: https://github.com/manjou/movie_api/blob/master/server.js#L208. 
These parameters passed to the API are also case-sensitive.

So it would be:-
` 
       data.Birthday = new Date(birthday).toISOString().split('T')[0]; 
`

Reply:
Additionally, you can do a similar thing for the password field - check if the user has entered a password before updating it.